### ISSUE TITLE
Optimizer is not available in AppCenter

### ISSUE THREAD

**Issue by JohnDumpling** [hannesschulze/optimizer#93]
After the release of eOS 6, Optimizer is not included in the AppCenter.

**Comment by pedro2555** [comment]
Yeah, this is terrible misjudgment from the eOS team. My understanding is, if it shows up it will be a flatpak.

Can one install this with apt?

**Comment by janxkoci** [comment]
I noticed there is not a single system monitor even on Flathub. I suspect that the flatpak container makes monitoring system resources tricky and apparently nobody still figured out how to do it right. Not even the GNOME folks with their system monitor.

**Comment by hannesschulze** [comment]
Hi, here's some more information on the current state of this app I wanted to share for some time now:

I have already attempted to package Optimizer as a flatpak a few months ago and was unable to figure out how to get the required permissions working. Packaging system monitors and similar programs as flatpaks might very well be possible, but I am not really sure about this because I don't have much experience with flatpak and I haven't tried it since then.

That being said, I have moved away from using elementary OS as my daily driver so I will probably not be able to get this working in the near future. I will probably update the main branch and the README to be compatible with elementary OS 6 in the next few days (so you can at least still use a manually compiled version if you want to) and then archive this repository for now to make this clear. Thank you for your interest in this project!
